src/Http/Auth/RouteRule.php

```diff
@@ -4,23 +4,37 @@
 
 namespace CultuurNet\UDB3\Http\Auth;
 
+use CultuurNet\UDB3\Http\Request\QueryParameters;
 use Psr\Http\Message\ServerRequestInterface;
 
 abstract class RouteRule
 {
     private string $pathPattern;
     private array $methods;
+    private ?string $excludeQueryParam;
 
-    public function __construct(string $pathPattern, array $methods)
+    public function __construct(string $pathPattern, array $methods, ?string $excludeGetParams = null)
     {
         $this->pathPattern = $pathPattern;
         $this->methods = $methods;
+        $this->excludeQueryParam = $excludeGetParams;
     }
 
     public function matchesRequest(ServerRequestInterface $request): bool
```

Comment:
With this extra change a unit test could be very useful.

Reply:
See PublicRouteRuleTest